packages/ihe-gateway/server/CodeTemplates/Common Library/AWS-Lambda-Invoke/AWS-Lambda-Invoke.js

```diff
@@ -47,9 +47,16 @@ function invokeLambda(functionName, payload) {
     const request = requestBuilder.build();
 
     const invokeResponse = lambdaClient.invoke(request);
-
     const responseString = invokeResponse.payload().asUtf8String();
     const responseJson = JSON.parse(responseString);
+
+    if (responseJson.statusCode != "200") {
+      throw new Error(
+        "Error invoking Lambda function: " +
+          functionName
+      );
```

Comment:
Can we log details about the error?`FunctionError`, `Payload`

[source](https://docs.aws.amazon.com/lambda/latest/api/API_Invoke.html#API_Invoke_ResponseElements)